project-management-infrastructure/src/main/java/life/qbic/projectmanagement/infrastructure/project/ProjectRepositoryImpl.java

```diff
@@ -90,7 +89,6 @@ public void update(Project project) {
   }
 
   @Override
-  @PostFilter("hasPermission(filterObject, 'READ')")
```

Comment:
What happened here???

Reply:
In this specific context checking the user permission makes no sense, since the user needs to be informed if his provided projectcode was used in ANY project, even the ones he doesn't have access to

Reply:
Please introduce a different method that checks for project exists instead. Otherwise you allow anybody to retrieve all project information for projects they do not have access to.

Reply:
Good point! I adapted the method accordingly to only retrieve a boolean indicating that the projectcode is already in use